lib/astronoby/nutation.rb

```diff
@@ -25,8 +25,8 @@ def for_ecliptic_longitude
         0,
         0,
         (
-          -17.2 * Math.sin(moon_ascending_node_longitude.to_radians.value) -
-          1.3 * Math.sin(2 * sun_mean_longitude.to_radians.value)
+          -17.2 * Math.sin(moon_ascending_node_longitude.radians) -
+          1.3 * Math.sin(2 * sun_mean_longitude.radians)
```

Comment:
Out of scope of this PR but there are a bunch of magic numbers that I have not clue what they do. Maybe extract some named constants?

Reply:
I forgot to answer to this one.
Most of the magic numbers in the library are magic numbers from the book I rely on. I don't always have explanation for these numbers, except that they are part of the formula.
I don't really know what to do with these. Magic numbers scream at me, but I can't invent names or use confusing names that might not be accurate.

Reply:
That's fair! I was hoping the book that provided the equation also provided an explanation of what these numbers were. Or at least gave their unit!